jsconfig-example.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
**Attn VSCode users:** If you don't already have a `jsconfig.json` file in the root of your project, create one and add the contents of this sample file to it. If you do already have a `jsconfig.json` file, update it with the relevant settings from this sample.

`jsconfig.json` is ignored via `.gitignore`, which is probably something we want to continue doing since the file can contain personal settings we don't want to share with everyone.